### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by djixas** [picotorrent/picotorrent#1245] [General]
Hi guys.

I'm migrated from qbittorrent and there doesn't seem to be any activity for over a year. Is the project dead and shall we migrate somewhere else?

**Comment by vktr** [comment]
Hi!

Yeah, there's been a year since the last release. I haven't really had energy to spend which is why development has stopped for a while. I don't think there's any critital open issues so I have just let things be the way they are.

So the project might have been inactive the last year, but it is not abandoned. I have a few major things I want to fix before doing a v1.0 release, I just haven't had time :smile:

**Reply by ghost** [reply]
Hi vktr, thank you for feedback!

**Comment by djixas** [comment]
Thank you, I think many would appreciate just a quarterly or so tweet on twitter, just so people stick with pico!

**Reply by vktr** [reply]
That's good feedback! I'll try to be more active :)

**Comment by ghost** [comment]
Hello everyone.

> Is the project dead? 

No. So I'm going to develop a resource for picotorrent, comment on this resource so I can check if it's something worth creating, developing for you. So please this: [(use case) orbit-db with picotorrent? ](https://github.com/picotorrent/picotorrent/discussions/1258) and [Choose location or storage provider in PicoTorrent](https://github.com/picotorrent/picotorrent/issues/1257)

**Comment by N0rdmann** [comment]
> I have a few major things I want to fix before doing a v1.0

How about another year later? 😇

Sry for bothering just was abt to switch from qBt on my win machine when i found this little project but when there are no more updates, at least with latest libtorrent i guess i stick with qBt. No offense intended, just asking if u may find ur motivation for this project again 🫶

**Reply by ClementNerma** [reply]
Even though there hasn't been any release since a bit more than 2 years, I still haven't any problem with it. I'm using as my main client and it's still working flawlessly :)

**Reply by rmarquis** [reply]
Dark theme support would deserve a point release by itself :)

**Comment by Sheppounet** [comment]
Hi ! @vktr First of all, thank you for your great projet.
It still works perfectly, but i'm asking the same question again, as few years goes and there is still no news. 
Is this project dead ?

**Reply by rmarquis** [reply]
It is safe to assume the project won't evolve anymore, unless forked and updated by someone else. 

I wish I succeeded in compiling the latest master code (for dark mode and latest fixes), but this has proven to be quite challenging.
